superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -475,6 +538,17 @@ export const ControlPanelsContainer = (props: ControlPanelsContainerProps) => {
           </Tabs.TabPane>
         )}
       </ControlPanelsTabs>
+      <div css={actionButtonsContainerStyles}>
```

Comment:
A super nit but I think this conflicts a little bit with the Emotion guidelines that we have. The goal of using `css` is to have it closer to the component where it's applied, especially if it's for a single use. In this case, you are using `css` but moving it far from the component to avoid taking too much space here I guess, so probably we should go with the standard `styled.div` instead